packages/runtime/src/helpers/edge.ts

```diff
@@ -470,6 +472,10 @@ export const writeEdgeFunctions = async ({
         ]),
       )
 
+      const dataRoutesMap = new Map(
```

Comment:
Can you add some comments for the changes in this file? It looks like we create a map of data routes (if data routes exist) and set the values to the regex corresponding to the data route, and later on below, it looks like the edge function gets mapped to the specific data route?

And for the cache part below, that's for edge caching but only if using the app dir feature?

Reply:
Sure thing. I'll add a comment. Re the caching, it's the same as the one right above: only enabled for appDir because it's experimental